### PR TITLE
Document std::hint::black_box

### PR DESCRIPTION
**Pull request by thomwiggers**

### CODE REVIEW

**Comment by thomwiggers** [comment]
Hopefully I did this correctly

File: data/1.66/hint_black_box.toml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+title = "`hint::black_box`"
+flag = "bench_black_box"
+rfc_id = 2630
```
Comment (robjtede) [inline data/1.66/hint_black_box.toml:3]:
```suggestion
rfc_id = 2360
```

**Review by robjtede — CHANGES_REQUESTED** [review]

**Review by robjtede — APPROVED** [review]

**Comment by robjtede** [comment]
Thanks 👍🏻

**Comment by jplatte** [comment]
Deployed!

**Pull request merged**